content/plotly/concepts/graph-objects/terms/scatterpolar/scatterpolar.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.Scatterpolar()'
+Description: 'Creates a scatter plot on polar axes.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Plotly'
+  - 'Graphs'
+  - 'Data'
+  - 'Values'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.Scatterpolar()`** function in Plotly is used to create scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes that are often used to represent periodic data. 
+
+## Syntax
+
+```pseudo
+plotly.graph_objects.Scatterpolar(x=None, y=None, ...)
+```           
+
+- `x`: A list of values to be used as the x-coordinates of the data points.  
+- `y`: A list of values to be used as the y-coordinates of the data points.  
```

Comment:
```suggestion
- `x`: A list of values to be used as the x-coordinates of the data points.  
- `y`: A list of values to be used as the y-coordinates of the data points.  
```
Here we should use `r` and `theta` for `plotly.graph_objects.Scatterpolar` because here we are dealing with polar coordinates, not the Cartesian coordinates, so here using z and y would be incorrect. So you can make the correction likewise -
r: It is a list of values representing the radial coordinates of the data points.
theta: It is a list of values representing the angular coordinates of the data points.

Reply:
duly noted with thanks